src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+
+    enum TaskType {
+        TODO, DEADLINE, EVENT
+    }
+    protected String description;
```

Comment:
There has been consistent use of camelCase for the variables in the PR which I like!